inference/worker/interface.py

```diff
@@ -1,24 +1,36 @@
-from typing import Literal
-
+from typing import List, Optional, Union
 import pydantic
-from oasst_shared.schemas import inference
+from oasst_shared import schemas
```

Comment:
I don't think we should change this, the entire codebase uses the same style of import we already use